samples/Build2019Demo.Unity/README.md

```diff
@@ -1,24 +1,49 @@
 # Build 2019 Demo
 
-This sample of using SpectatorView is very similar to the live code demo presented at //BUILD 2019 conference ([video link](https://www.youtube.com/watch?v=P8og3nC5FaQ)).
+This sample of using SpectatorView is very similar to the live code demo presented at //BUILD 2019 conference.
 
-## Contents
+[![//BUILD 2019 Video](../../doc/images/Build2019DemoVideo.png)](https://www.youtube.com/watch?v=P8og3nC5FaQ&t=2255 "//BUILD 2019 Video")
+
+## Running the Demo
+
+In order to run the demo, you will need at least two MR/AR capable devices, with the host device preferably a HoloLens 2. Once you have the devices ready, follow the instructions below.
+
+> **ARKit Note:** If you wish to run the experience on an iOS device with ARKit, download the [ARKit repository](https://bitbucket.org/Unity-Technologies/unity-arkit-plugin/downloads/), unzip it, and copy the contents of the `Unity-Technologies-unity-arkit-plugin-94e47eae5954/Assets` folder to `/samples/Build2019Demo.Unity/Assets/`.
+
+### Instructions
+
+These instructions assumes a HoloLens 2 device for host, and an Android phone for spectator. For iOS, replace references to Android with iOS.
```

Comment:
nit: I think host should be replaced with user here to remain consistent with the enum definition.